app/views/preferences/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :heading do %>
+  <h1><%= t ".title" %></h1>
+<% end %>
+
+<dl class="row">
+  <dt class="col-sm-4"><%= t ".preferred_editor" %></dt>
+
+  <% if current_user.preferred_editor? %>
+    <dd class="col-sm-8"><%= t("editor.#{current_user.preferred_editor}.description") %></dd>
+  <% else %>
+    <dd class="col-sm-8"><%= t("editor.default", :name => t("editor.#{Settings.default_editor}.name")) %></dd>
+  <% end %>
+
+  <dt class="col-sm-4"><%= t ".preferred_languages" %></dt>
+
+  <dd class="col-sm-8">
+    <ul>
+      <% current_user.preferred_languages.each do |locale| %>
+        <li><%= locale %></li>
```

Comment:
I thought I had mentioned this in my previous review but apparently I missed it...

I don't know about you but I'm not particularly keen on the bulleted list here... What do you think to just a simple list? (I did `list-style: none; padding-left: 0` to test it)

Reply:
Done.